server/src/main/java/org/opentosca/toscana/core/plugin/TOSCAnaPlugin.java

```diff
@@ -1,155 +0,0 @@
-package org.opentosca.toscana.core.plugin;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
-
-import org.opentosca.toscana.core.plugin.lifecycle.ExecutionPhase;
-import org.opentosca.toscana.core.plugin.lifecycle.TransformationLifecycle;
-import org.opentosca.toscana.core.plugin.lifecycle.ValidationFailureException;
-import org.opentosca.toscana.core.transformation.TransformationContext;
-import org.opentosca.toscana.core.transformation.platform.Platform;
-import org.opentosca.toscana.core.util.Lifecycle;
-import org.opentosca.toscana.core.util.LifecyclePhase;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- This class represents the base class for every Plugin that wants to implement the Transformation lifecycle
-
- @param <LifecycleT> The class type of the plugin specific implementation of the Lifecycle interface */
-public abstract class TOSCAnaPlugin<LifecycleT extends TransformationLifecycle> implements Lifecycle {
```

Comment:
I honestly dont like this whole renaming thing. The original file was written by me. even though you changed it many lines would still be mine. After that ALL lines belong to you and i honestly dont like that. Als why should we rename this? Just because of some very "important" naming convention?
![image](https://user-images.githubusercontent.com/2706976/35865808-ce1c38c4-0b55-11e8-877d-dd34fbb6aa46.png)

Reply:
I did not intend to 'steal' lines. I simply keep refactoring code all the time.
I'm trying to roll back the changes..

Reply:
In addition to renaming, the file got moved in order to make use of package visibility. This caused the change of the line ownership. git-blame's `-C` flag solves your issue. Is that ok? We have to keep on refactoring the codebase...